hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/wal/AbstractFSWAL.java

```diff
@@ -211,6 +212,8 @@
    */
   protected final int maxLogs;
 
+  protected final boolean useHsync;
```

Comment:
I always get these confused? You can't just call it sync? Whats diff between hsync, sync, and fsync again? I forgot?